pkg/storage/leveldb/leveldb_store.go

```diff
@@ -126,3 +128,11 @@ func (s *leveldbStore) Get(k string) ([]byte, error) {
 	}
 	return data, nil
 }
+
+// Iterator returns iterator for the latest snapshot of the underlying db.
+func (s *leveldbStore) Iterator(start, limit string) storage.StoreIterator {
+	if start == "" || limit == "" {
+		iterator.NewEmptyIterator(errors.New("start key is mandatory"))
```

Comment:
wrong error message `start key is mandatory` better to change it `start or limit key is mandatory` or smth similar

Reply:
done